### DISCUSSION TITLE
http://localhost:3000/blog/2024/02/27/improvements-rust-translation

### DISCUSSION THREAD

**Discussion by giscus** [formal-land/formal.land#12] [Blog comments]
# http://localhost:3000/blog/2024/02/27/improvements-rust-translation

Our tool coq-of-rust is translating Rust&nbsp;🦀 programs to the proof system Coq&nbsp;🐓 to do formal verification on Rust programs. Even if we are able to verify realistic code, such as as ERC-20 smart contract, coq-of-rust still has some limitations:

http://localhost:3000/blog/2024/02/27/improvements-rust-translation

**Comment by clarus** [comment]
First test of comment!!!
